ext/CodecZstdExt.jl

```diff
@@ -0,0 +1,54 @@
+module CodecZstdExt
+
+using Zarr: Zarr
+using JSON: JSON
+using CodecZstd: CodecZstd
+
+"""
+    ZstdZarrCompressor(clevel::Int=0)
+    ZstdZarrCompressor(c::CodecZstd.ZstdCompressor, [d::CodecZstd.ZstdDecompressor])
+
+Zstandard compression for Zarr.jl. This is a `Zarr.Compressor` wrapper around
+`CodecZstd`. Construct with either the compression level, `clevel`, or by
+providing an instance of a `ZstdCompressor`.  `ZstdFrameCompressor` is
+recommended.
+"""
+struct ZstdZarrCompressor <: Zarr.Compressor
+    compressor::CodecZstd.ZstdCompressor
+    decompressor::CodecZstd.ZstdDecompressor
+end
```

Comment:
This doesn't support multithreaded use IIUC. I think this should be like https://github.com/JuliaIO/Zarr.jl/blob/f436713a10a9d16ca82417176e1c8f6a3e26bc8a/src/Compressors.jl#L129-L131 where the struct only contains the parameters of the codec.